<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактирование сеанса</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(122, 122, 122);</string>
  </property>
  <widget class="QCalendarWidget" name="calendar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>392</width>
     <height>236</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 40, 40);
alternate-background-color: rgb(56, 0, 0);
color: rgb(255, 85, 127);
selection-background-color: rgb(175, 0, 87);
</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_films">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>200</y>
     <width>431</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(141, 19, 37);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Фильм:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Дата:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_time">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>50</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(141, 19, 37);</string>
   </property>
   <item>
    <property name="text">
     <string>9:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10:30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13:30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16:30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19:30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21:00</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22:30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0:00</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Время:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="box_rooms">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(141, 19, 37);</string>
   </property>
   <item>
    <property name="text">
     <string>Малый зал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Средний зал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Большой зал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Vip зал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Imax зал</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Зал:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_price">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>321</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(141, 19, 37);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Цена билета:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
</string>
   </property>
   <property name="text">
    <string>руб.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>501</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(104, 0, 62);</string>
   </property>
   <property name="text">
    <string>Подтвердить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
